### PR TITLE
feature/SOF-3765-1

### PR DESCRIPTION
**Pull request by mohammadidinani**

### CODE REVIEW

File: editor/js/commands/RemoveObjectCommand.js [hunk 1]
```diff
@@ -29,6 +29,8 @@ RemoveObjectCommand.prototype = {
 
 	execute: function () {
 
+		if (["PerspectiveCamera", "AmbientLight", "DirectionalLight"].includes(this.object.type)) return;
```
Comment (timurbazhirov) [inline editor/js/commands/RemoveObjectCommand.js:33]: Inline comment

File: editor/js/Sidebar.Scene.js [hunk 2]
```diff
@@ -202,9 +202,11 @@ Sidebar.Scene = function ( editor ) {
 
 				var object = objects[ i ];
 
-				var option = buildOption( object, true );
-				option.style.paddingLeft = ( pad * 10 ) + 'px';
-				options.push( option );
+				if (!["PerspectiveCamera", "AmbientLight", "DirectionalLight"].includes(object.type)) {
```
Comment (timurbazhirov) [inline editor/js/Sidebar.Scene.js:206]: Inline comment

File: editor/js/Sidebar.Geometry.LineSegments.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
+Sidebar.Geometry.LineSegments = function (editor, object) {
```
Comment (timurbazhirov) [inline editor/js/Sidebar.Geometry.LineSegments.js:4]: Docstring?

File: editor/js/Menubar.Edit.js [hunk 4]
```diff
@@ -195,7 +195,7 @@ Menubar.Edit = function ( editor ) {
 				" material(s) were changed.\n" + errors.join( "\n" ) );
 
 	} );
-	options.add( option );
+//	options.add( option );
```
Comment (timurbazhirov) [inline editor/js/Menubar.Edit.js:198]: Is there a way to pass a config and then enable or disable these items based on it? That way we can put up a PR to original repo.

File: editor/js/Sidebar.Geometry.LineSegments.js (hunk 3 above)
Comment (mohammadidinani) [reply editor/js/Sidebar.Geometry.LineSegments.js:4]: Added.

File: editor/js/Sidebar.Scene.js (hunk 2 above)
Comment (mohammadidinani) [reply editor/js/Sidebar.Scene.js:206]: Added.

File: editor/js/commands/RemoveObjectCommand.js (hunk 1 above)
Comment (mohammadidinani) [reply editor/js/commands/RemoveObjectCommand.js:33]: Added.

**Pull request merged**